server/src/server.ts

```diff
@@ -763,6 +793,9 @@ function onMessage(msg: m.Message) {
     } else if (msg.method === DidCloseTextDocumentNotification.method) {
       let params = msg.params as p.DidCloseTextDocumentParams;
       closedFile(params.textDocument.uri);
+    } else if (msg.method === DidChangeConfigurationNotification.type.method) {
```

Comment:
should this be `.type.method` or `.method` like in the other cases?

Reply:
According to the types, this notification does not have `.method`, only `.type.method`.